### ISSUE TITLE
problem with async Storage multiple connections after reload

### ISSUE THREAD

**Issue by duilio92** [infinitered/reactotron#1211]
Whenever I re load my app a new connection is created (as if it were a different device).

Here is my Reactotron initial configuration:

```javascript
import {AsyncStorage} from '@react-native-async-storage/async-storage';
import Config from './Config/DebugConfig';

if (Config.useReactotron){
    Reactotron
        .setAsyncStorageHandler(AsyncStorage) // AsyncStorage would either come from `react-native` or `@react-native-community/async-storage` depending on where you get it from
        .configure() // controls connection & communication settings
        .useReactNative() // add all built-in react native plugins
        .connect() // let's connect!
```

I can't figure out what is wrong

**Comment by juddey** [comment]
did you figure this out? Its started happening to me.

**Comment by Ajmal0197** [comment]
Use it like this will create only one connection:
`import AsyncStorage from '@react-native-async-storage/async-storage';

Reactotron.configure({ name: 'app_name' })
  .setAsyncStorageHandler(AsyncStorage)
  .useReactNative()
  .use(sagaPlugin())
  .use(reactotronRedux()) //  <- here i am!
  .connect();
`

**Comment by cuantmac** [comment]
@Ajmal0197 you are right! I resolved this issue with .setAsyncStorageHandler(AsyncStorage)

**Comment by fikkatra** [comment]
In my case, after adding `setAsyncStorageHandler(AsyncStorage)` to `ReactotronConfig` there were indeed 2 connections. However, after quitting and restarting Reactotron, then reloading the app, it only showed 1 connection.
